### PR TITLE
[embeddings] Update descriptions for 2 SVs

### PR DESCRIPTION
**Pull request by chejennifer**
- add "population by age" for the AgeDistribution topic
     - This was added to promote the AgeDistribution topic to the top in cases like "population by age in the United States". 
     - before: https://screenshot.googleplex.com/AW5eV5DXrVD4cDe
     - after: https://screenshot.googleplex.com/6MNeE4PDrsuProY
- add "gender of population" for the Gender topic
     - This was added to promote the Gender topic to the top in cases like "population by gender in the United States". 
     - before: https://screenshot.googleplex.com/4fvN5rk5HbNobh7
     - after: https://screenshot.googleplex.com/4yT2P3o86Yq2jHc

validation: https://pantheon.corp.google.com/storage/browser/_details/datcom-embedding-diffs/chejennifer_medium_ft_2023_12_11_22_30_00.html

### CODE REVIEW

File: server/integration_tests/test_data/detection_api_statvars/correlatewithgdpofcalifornia/chart_config.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "dc/topic/Economy",
     "dc/topic/sdg_17.1.1",
     "dc/topic/EconomicEquity",
-    "dc/topic/GDP",
-    "Amount_EconomicActivity_GrossDomesticProduction_RealValue"
+    "Amount_EconomicActivity_GrossDomesticProduction_RealValue",
```
Comment (shifucun) [inline server/integration_tests/test_data/detection_api_statvars/correlatewithgdpofcalifornia/chart_config.json:44]: why are these two swapped?

File: tools/nl/embeddings/data/curated_input/sheets_svs.csv [hunk 2]
```diff
@@ -1520,7 +1520,7 @@ dc/topic/Emissions_PM2.5_EmissionsbySource,Emissions of PM2.5 By Source,Emission
 dc/topic/Energy_Generation_FossilFuels,Energy Generation From Fossil Fuels,Energy Generation From Fossil Fuels,,energy produced from fossil fuels;
 dc/topic/Fire,Fires,Fires,,fire; fire event; wild fires;
 dc/topic/GreenhouseGasEmissions,Greenhouse Gas Emission,Greenhouse Gas Emissions,,GHG emissions; greenhouse gasses emissions; sources of greenhouse gas emissions; sources of GHG emissions;
-dc/topic/Gender,Gender,Gender,,
+dc/topic/Gender,Gender,Gender,,gender of population;
```
Comment (shifucun) [inline tools/nl/embeddings/data/curated_input/sheets_svs.csv:1523]: what does ";" mean in this csv?

File: server/integration_tests/test_data/detection_api_statvars/correlatewithgdpofcalifornia/chart_config.json (hunk 1 above)
Comment (chejennifer) [reply server/integration_tests/test_data/detection_api_statvars/correlatewithgdpofcalifornia/chart_config.json:44]: looks like this is the order on autopush so I don't think it's a change from this PR:

https://screenshot.googleplex.com/5JVV9RjsDTiw2Dv

File: tools/nl/embeddings/data/curated_input/sheets_svs.csv (hunk 2 above)
Comment (chejennifer) [reply tools/nl/embeddings/data/curated_input/sheets_svs.csv:1523]: ";" is the separator for within each csv cell

Comment (jehangiramjad) [reply tools/nl/embeddings/data/curated_input/sheets_svs.csv:1523]: in other words, a semi colon is used as a separator between alternative sentences/descriptions etc.

**Review by jehangiramjad — COMMENTED** [review]
Thanks. Two meta comments:

1. In the PR description, could you also state why this was needed in the first place, i.e. what specific scenarios are you trying to support?

2. Could you also have screenshots of example(s) where the new strings (or some very similar strings to the new ones being added) are now showing the "right" result on the NL results (charts + debug)? A before and after screenshot pairing would be great!

**Comment by chejennifer** [comment]
> Thanks. Two meta comments:
> 
> 1. In the PR description, could you also state why this was needed in the first place, i.e. what specific scenarios are you trying to support?
> 2. Could you also have screenshots of example(s) where the new strings (or some very similar strings to the new ones being added) are now showing the "right" result on the NL results (charts + debug)? A before and after screenshot pairing would be great!

done. Also let me know if you think there are better ways to promote those 2 charts

**Comment by jehangiramjad** [comment]
> > Thanks. Two meta comments:
> > 
> > 1. In the PR description, could you also state why this was needed in the first place, i.e. what specific scenarios are you trying to support?
> > 2. Could you also have screenshots of example(s) where the new strings (or some very similar strings to the new ones being added) are now showing the "right" result on the NL results (charts + debug)? A before and after screenshot pairing would be great!
> 
> done. Also let me know if you think there is a better way to promote those 2 charts

I think approach works. I'd recommend adding more than just a single alternative (for future PRs). For example, in addition to "gender of population" you could also add "population by gender; gender distribution; gender splits;" etc

**Review by jehangiramjad — APPROVED** [review]

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**